brave/browser-laptop
# Removes transition btc->bat code

Resolves #12712

## Do not squash commits

## Submitter Checklist:


## Test Plan:

1) old wallet
- create wallet on version 0.18 so that you have BTC wallet
- copy that wallet into this branch
- run browser
- check payments page (you should see new wallet created for you)
- check profile folder to see if there is file `ledger-state-btc.json`

2) new wallet
- create wallet on 0.20
- add some BAT into it
- copy that wallet into this branch
- run browser
- make sure that no money was lost and that old wallet is still there


## Reviewer Checklist:


Tests
